save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityTableView.kt

```diff
@@ -0,0 +1,354 @@
+            isUserProfilePage -> VulnerabilityFilter(authorName = props.userName)
+            isOrganizationPage -> VulnerabilityFilter(organizationName = props.organizationName)
+            else -> VulnerabilityFilter.empty
+        }
+    )
+    val (countVulnerability, setCountVulnerability) = useState(0)
+
+    val (selectedMenu, setSelectedMenu) = useState(VulnerabilityListTab.PUBLIC)
+    val (isOwnerTab, setIsOwnerTab) = useState(false)
+
+    val enrollRequest = useDeferredRequest {
+        val count: Int = post(
+            url = "$apiUrl/vulnerabilities/count/by-filter",
+            headers = jsonHeaders,
+            body = Json.encodeToString(vulnerabilityFilter),
+            loadingHandler = ::noopLoadingHandler,
+            responseHandler = ::noopResponseHandler,
+        ).unsafeMap {
+            it.decodeFromJsonString()
+        }
+        setCountVulnerability(count)
+    }
+
+    val vulnerabilityListTable: FC<VulnerabilityListTableProps> = useMemo(isVulnerabilityCollectionPage, isUserProfilePage, isOrganizationPage, isOwnerTab, i18n.language) {
+        tableComponent(
+            columns = {
+                columns {
+                    column(id = "identifier", header = "Identifier".t(), { identifier }) { cellContext ->
+                        Fragment.create {
+                            td {
+                                className = ClassName("align-middle text-nowrap")
+                                div {
+                                    className = ClassName("row m-0")
+                                    Link {
+                                        to = "/${FrontendRoutes.VULNERABILITY_SINGLE}/${cellContext.row.original.identifier}"
+                                        +cellContext.value
+                                    }
+                                }
+                                div {
+                                    className = ClassName("row m-0")
+                                    cellContext.row.original.tags.forEach { tag ->
+                                        div {
+                                            className = ClassName("rounded-pill px-1 mt-1 mr-1")
+                                            style = jso {
+                                                fontSize = 0.8.rem
+                                                color = Colors.VULN_PRIMARY.value.unsafeCast<ColorProperty>()
+                                                border = "0.01rem solid $color".unsafeCast<Border>()
+                                            }
+                                            +tag
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    column(id = "short_description", header = "Description".t(), { summary }) { cellContext ->
```

Comment:
can we rename `id` and `header` to `Summary` to have this field as in COSV?

Reply:
Done